lib/Dialect/Comb/CombFolds.cpp

```diff
@@ -1121,5 +1270,14 @@ LogicalResult ICmpOp::canonicalize(ICmpOp op, PatternRewriter &rewriter) {
     }
   }
 
+  if (auto lhs = dyn_cast_or_null<ConcatOp>(op.lhs().getDefiningOp())) {
```

Comment:
Plz add a comment describing the transformation here.

Reply:
Done